src/Apps/Auction/Components/__tests__/RegistrationForm.test.tsx

```diff
@@ -26,7 +26,7 @@ describe("RegistrationForm", () => {
     trackSubmissionErrors: jest.fn(),
   }
 
-  beforeAll(() => {
+  beforeEach(() => {
```

Comment:
was there a bug latent within these tests? what motivated this change?

Reply:
We started adding a test here and noticed some leakage caused by this not being called before each spec. I left it in because it does fix that in case someone ends up adding more specs here.